buildspec.yml

```diff
@@ -21,7 +21,6 @@ phases:
       # run local tests
       - tox -e py36 -- test/integration/local --framework-version 1.11
       - tox -e py36 -- test/integration/local --framework-version 1.12
-      - tox -e py36 -- test/integration/local --framework-version 1.13
```

Comment:
is this spec for PR builds or release?

Can we add 1.13 and 1.14 for PR builds ? Let us know if there are any changes needed for dockerfiles to enable PR builds

Reply:
Yes. The PR builds run all integ tests. We are not adding 1.13 and 1.14 to PR builds because as we talked offline, we are currently not adding tensorflow dependency in this container. In that case, some tests will fail.